.github/pull_request_template.md

```diff
@@ -1,18 +1,20 @@
+# Description
+
 <!--
 What is the purpose of this PR?
 
-* What is the context surrounding this PR? Include links if possible.
-* What kind of feedback do you want?
 * Have you [formatted the PR title](https://github.com/pinterest/gestalt/#releasing)? `ComponentName: Description`
 -->
 
-## Test Plan
+## Links
 
-<!--
-How can reviewers verify this is good to merge?
+- JIRA: https://jira.pinadmin.com/browse/PDS-XXX
+- [TDD](paper doc link)
+- [Figma](link to figma file)
 
-* Is it tested?
-* Is it accessible?
-* Is it documented?
-* Have you involved other stakeholders (such as a Pinterest Designer)?
--->
+## Test Plan
+
+- [ ] Added Unit Tests
+- [ ] Added Accessibility Tests
+- [ ] Visually checked for regressions
+- [ ] Stakeholder feedback (ex. Gestalt designers)
```

Comment:
_[GOOD] Accessibility: validate keyboard & screen reader interaction
_[GOOD] Browser Support: test in multiple browsers (Chrome / Safari / Firefox)
_[GOOD] Dark Mode: view your component in dark mode
_[GOOD] Documentation: document your component & provide examples
_[GOOD] Responsiveness: view your component in sm/md/lg screen sizes
_[GOOD] RTL: Right to Left support
_[GOOD] Flow Tests
_[GOOD] Unit Tests

Is it worth incorporating any of the ones in the TDD in here? for instance...

[  ] Added/Updated Documentation
[  ] Checked for RTL support
etc.